types/msg.go

```diff
@@ -64,10 +64,12 @@ type EventAttribute struct {
 // CosmosMsg is an rust enum and only (exactly) one of the fields should be set
 // Should we do a cleaner approach in Go? (type/data?)
 type CosmosMsg struct {
-	Bank    *BankMsg        `json:"bank,omitempty"`
-	Custom  json.RawMessage `json:"custom,omitempty"`
-	Staking *StakingMsg     `json:"staking,omitempty"`
-	Wasm    *WasmMsg        `json:"wasm,omitempty"`
+	Bank     *BankMsg        `json:"bank,omitempty"`
+	Custom   json.RawMessage `json:"custom,omitempty"`
+	Ibc      *IbcMsg         `json:"ibc,omitempty"`
```

Comment:
Consistent naming for IBC: Sometimes the type prefix is uppercase `IBC` but here and some other places `Ibc`. Personally I prefer `IBC` but whatever naming should be consistent in the project.

Reply:
All Rust code should be strict camel case, including the acronyms. For the Go side of things, I'm open to adapt to the caller's naming convention.

Reply:
Good point here.The question is language linters for acronyms/abreviations.

Rust linter enforces eg. `ContractId`.
Go linter enforce eg `ContractID`

I was trying to maintain this with `Ibc` on rust and `IBC` in go. This is one place where I slipped.
Unless there are objections I will keep it different in both languages, but consistent within each language